src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -323,10 +336,17 @@ export function SavedObjectProvider(Promise, Private, Notifier, confirmModalProm
     };
 
     const displayDuplicateTitleConfirmModal = () => {
-      const confirmMessage =
-        `A ${this.getDisplayName()} with the title '${this.title}' already exists. Would you like to save anyway?`;
+      const confirmMessage = i18n.translate('common.ui.courier.savedObject.confirmModal.saveDuplicateConfirmationMessage', {
+        defaultMessage: 'A {name} with the title \'{title}\' already exists. Would you like to save anyway?',
```

Comment:
Please use a template literal here (backtick string) to prevent the need of escaping the single quote.